cypress/integration/vanilla-opensearch-dashboards/dashboard_sample_data_spec.js

```diff
@@ -0,0 +1,104 @@
+
+/**
+ * dashboard_sample_data test suite description:
+ * 1) Visit the home page of opensearchdashboard, check key UI elements display
+ * 2) add sample data of eCommerce, flights, web logs from tutorial page
+ * 3) check each sample data dashboard key UI elements display
+ */
+
+const testFixtureHandler = new TestFixtureHandler(cy, Cypress.env('openSearchUrl'))
+const commonUI = new CommonUI(cy)
+const dashboardPage = new DashboardPage(cy)
+const miscUtils = new MiscUtils(cy)
+describe('dashboard sample data validation', () => {
+    before(() => {
+
+    })
+
+    after(() => {
+
+    })
+
+    describe('checking home page', () => {
+        before(() => {
+            // Go to the home page
+            miscUtils.visitPage('app/home#')
+        })
+
+        after(() => {
+
+        })
+
+        it('checking opensearch_dashboards_overview display', () => {
+            // Check that opensearch_dashboards_overview is visable
+            commonUI.checkElementExists('a[href="/_dashboards/app/opensearch_dashboards_overview"]', 1)
+        })
+
+        it('checking tutorial_directory display', () => {
+            // Check that tutorial_directory is visable
+            commonUI.checkElementExists('a[href="/_dashboards/app/home#/tutorial_directory"]', 2)
+        })
+
+        it('checking management display', () => {
+            // Check that management is visable
+            commonUI.checkElementExists('a[href="/_dashboards/app/management"]', 1)
+        })
+
+        it('checking dev_tools display', () => {
+            // Check that dev_tools is visable
+            commonUI.checkElementExists('a[href="/_dashboards/app/dev_tools#/console"]', 2)
+        })
+
+        it('settings display', () => {
+            // Check that settings is visable
+            commonUI.checkElementExists('a[href="/_dashboards/app/management/opensearch-dashboards/settings#defaultRoute"]', 1)
+        })
+
+        it('checking feature_directory display', () => {
+            // Check that feature_directory is visable
+            commonUI.checkElementExists('a[href="/_dashboards/app/home#/feature_directory"]', 1)
+        })
+
+        it('checking navigation display', () => {
+            // Check that navigation is visable
+            commonUI.checkElementExists('button[data-test-subj="toggleNavButton"]', 1)
+        })
+
+        it('checking Help menu display', () => {
+            // Check that Help menu is visable
+            commonUI.checkElementExists('button[aria-label="Help menu"]', 1)
+        })
+
+    })
+
+    describe('adding sample data', () => {
+        before(() => {
+            miscUtils.addSampleData()
+        })
+
+        after(() => {
+            miscUtils.removeSampleData()
+        })
+
+        it('checking ecommerce dashboards displayed', () => {
+            miscUtils.viewData('ecommerce')
+            commonUI.checkElementContainsValue('span[title="\[eCommerce\] Revenue Dashboard"]', 1, '\\[eCommerce\\] Revenue Dashboard')
```

Comment:
move ``div[data-test-subj="markdownBody"] > h3`` to page model library or at least make it constants for readability and maintainability 

e.g.
```
const dashboards_homepage_xxx_dashboards_yyy_label_element_xpath
```

Reply:
will change this